packages/experience/src/apis/experience/interaction.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  type InteractionEvent,
+  type IdentificationApiPayload,
+  type UpdateProfileApiPayload,
+} from '@logto/schemas';
+
+import api from '../api';
+
+import { experienceApiRoutes } from './const';
+
+type SubmitInteractionResponse = {
+  redirectTo: string;
+};
+
+export const initInteraction = async (interactionEvent: InteractionEvent) =>
+  api.put(`${experienceApiRoutes.prefix}`, {
+    json: {
+      interactionEvent,
+    },
+  });
+
+export const identifyUser = async (payload: IdentificationApiPayload = {}) =>
+  api.post(experienceApiRoutes.identification, { json: payload });
+
+export const submitInteraction = async () =>
+  api.post(`${experienceApiRoutes.submit}`).json<SubmitInteractionResponse>();
+
+export const _updateProfile = async (payload: UpdateProfileApiPayload) =>
```

Comment:
why `_`?

Reply:
This is for private internal API reference use only. Have another `updateProfile` API exported to the web components. 

<img width="634" alt="image" src="https://github.com/user-attachments/assets/37acfee9-67f6-4b1e-aa76-141c62afb94b">

Reply:
Updated change the API name:

`_updateProfile` -> `updateProfile`
`updateProfile` -> `fulfillProfile`